notebooks/official/tensorboard/vertex_tensorboard_custom_training_with_custom_container.ipynb

```diff
@@ -0,0 +1,900 @@
+{
```

Comment:
<p>Suggest to incorporate a Technical Writer to review the text.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/611/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>

Reply:
<p>I almost just copied from the official doc here: https://screenshot.googleplex.com/8yaC9Z6ioBKxQQL</p>